tests/managed_services/mas_debezium/consumer_pod.py

```diff
@@ -0,0 +1,69 @@
+from ocp_resources.constants import TIMEOUT_4MINUTES
+from ocp_resources.pod import Pod
+
+
+class ConsumerPod(Pod):
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        client=None,
+        teardown=True,
+        privileged_client=None,
+        yaml_file=None,
+        consumer_image=None,
+        kafka_bootstrap_url=None,
+        kafka_sa_client_id=None,
+        kafka_sa_client_secret=None,
+        kafka_test_topic=None,
+        delete_timeout=TIMEOUT_4MINUTES,
+        **kwargs,
+    ):
+        super().__init__(
+            name=name,
+            namespace=namespace,
+            client=client,
+            teardown=teardown,
+            privileged_client=privileged_client,
+            yaml_file=yaml_file,
+            delete_timeout=delete_timeout,
+            **kwargs,
+        )
+        self.consumer_image = consumer_image
+        self.kafka_bootstrap_url = kafka_bootstrap_url
+        self.kafka_sa_client_id = kafka_sa_client_id
+        self.kafka_sa_client_secret = kafka_sa_client_secret
+        self.kafka_test_topic = kafka_test_topic
+
+    def to_dict(self):
+        super().to_dict()
+
+        self.res.update(
+            {
+                "spec": {
+                    "containers": [
+                        {
+                            "name": self.name,
+                            "image": self.consumer_image,
+                            "command": [
+                                "kcat",
+                                "-b",
+                                self.kafka_bootstrap_url,
+                                "-X",
+                                "sasl.mechanisms=PLAIN",
+                                "-X",
+                                "security.protocol=SASL_SSL",
+                                "-X",
+                                f"sasl.username={self.kafka_sa_client_id}",
+                                "-X",
+                                f"sasl.password={self.kafka_sa_client_secret}",
+                                "-t",
+                                self.kafka_test_topic,
+                                "-C",
+                                "-e",
```

Comment:
1. please use shlex to convert the command string to a list
2. this should be use in pod.execute not here